s3-spark/src/test/scala/geotrellis/spark/io/s3/util/S3RangeReaderProviderSpec.scala

```diff
@@ -1,14 +1,18 @@
 package geotrellis.spark.store.s3.util
 
+import geotrellis.spark.store.s3.S3TestUtils
```

Comment:
I'm wondering, should `S3TestUtils` live in the `testkit` or not? @moradology

Reply:
The good news is that there is no longer an s3-testkit to put it in